src/main/java/com/example/songr/controllers/AlbumController.java

```diff
@@ -27,9 +35,51 @@
         album.imageURL = "https://en.wikipedia.org/wiki/Led_Zeppelin_III#/media/File:Led_Zeppelin_-_Led_Zeppelin_III.png";
 
         this.repo.save(album);
+        //Returns all the albums including the new one
 
-        Iterable<Album> albums = this.repo.findAll();
-
-        return albums;
+        return "albums";
     }
+
+//    @GetMapping("/album/{id}")
+//    public Album getAlbum(@PathVariable Long id) {
+//    //Getting the Album by id
+//        Optional<Album> album = this.repo.findById(id);
+//        if (album.isPresent()) {
+//            //return value of Album Optional
+//            return album.get();
+//        } else {
+//            throw new AlbumNotFoundException();
+//        }
+//    }
+//
+//    @PostMapping("/album")
+//    public Album createAlbum(@RequestBody Album album){

```

Comment:
For today's lab, you'll need to accept each item as a (@RequestParam String title, @RequestParam int length) then create a new album manually (i.e. Album a1 =  new Album(title, length);) then it will save it to the DB.